easybuild/easyconfigs/t/triqs_dfttools/triqs_dfttools-2.2.0-foss-2019a-Python-2.7.15.eb

```diff
@@ -0,0 +1,68 @@
+easyblock = 'CMakeMake'
+
+name = 'triqs_dfttools'
```

Comment:
`DFTTools`?

Reply:
This should in fact be `trqis_dft_tools` if we want to match our ubuntu package name
`https://triqs.github.io/dft_tools/latest/install.html#packaged-versions-of-dfttools`. The file would have to be renamed accordingly of course